src/sync.rs

```diff
@@ -937,10 +929,8 @@ QT test123: Original text"
 
         let tweets = Vec::new();
         let toots = vec![boost];
-        let options = SyncOptions {
-            sync_reblogs: false,
-            sync_retweets: true,
-        };
+        let mut options = get_sync_options();
+        options.sync_reblogs = false;
```

Comment:
An alternative I considered: Instead of changing all tests to use `get_sync_options()` instead of `DEFAULT_SYNC_OPTIONS`, maybe I can instead use `clone` only in the tests that need to modify the default values. Something along the following lines:

```rust
let mut options = DEFAULT_SYNC_OPTIONS.clone();
options.sync_reblogs = false;
```

What do you think?

Reply:
I went ahead and implemented this simpler version. The patch is much smaller now 🕺